traffic_ops/testing/api_contract/v4/conftest.py

```diff
@@ -436,10 +436,10 @@ def cachegroup_post_data(to_session: TOSession, request_template_data: list[JSON
 		if not isinstance(name, str):
 			raise TypeError(f"name must be str, not '{type(name)}'")
 		cachegroup["name"] = name[:4] + randstr
-		short_name = cachegroup["shortName"]
+		short_name = cachegroup["shortname"]
```

Comment:
Why are you changing shortName to shortname, because of this test_cachegroup TC is failing. Refer to request_template.json , It is mentioned as shortName not shortname.

Reply:
When I ran in the pylinter it said that it didn't follow snake casing.

Reply:
that's a string index, not a property access or variable declaration, so Pylint should not be checking it like that.

Reply:
I changed it back. There was an external extension giving me that snake casing error.